__tests__/pages/dashboard.test.js

```diff
@@ -77,6 +96,10 @@ describe('Dashboard', () => {
     expect(cppdResult).toBeInTheDocument()
   })
 
+  it('renders NoBenefitCards', () => {
+    expect(NoBenefitCards).toBeTruthy()
```

Comment:
Unfortunately all this is doing is testing the code from the const, which is in the test file. If the dashboard changes this will remain completely un-effected. The const NoBenefitCards really isn't needed.